examples/30_extended/fetch_runtimes.py

```diff
@@ -0,0 +1,132 @@
+# ***************************************
+
+task_id = 1792  # 1778  # supervised classification on mfeat-fourier
+task = openml.tasks.get_task(task_id)
+print(task)
+
+# Viewing associated data
+X, y = task.get_X_and_y(dataset_format="dataframe")
+print(X.head)
+n_repeats, n_folds, n_samples = task.get_split_dimensions()
+print(
+    "Task {}: number of repeats: {}, number of folds: {}, number of samples {}.".format(
+        task_id, n_repeats, n_folds, n_samples,
+    )
+)
+
+clf = RandomForestClassifier(n_estimators=10)
+
+# Creating utility function
+def print_compare_runtimes(measures):
+    for repeat, val1 in measures["usercpu_time_millis_training"].items():
+        for fold, val2 in val1.items():
+            print(
+                "Repeat #{}-Fold#{}: CPU-{:.3f} vs Wall-{:.3f}".format(
+                    repeat, fold, val2, measures["wall_clock_time_millis_training"][repeat][fold]
+                )
+            )
+        print()
+
+
+############################################################################
+# Case 1: Running a Random Forest model on an OpenML task
+# *******************************************************
+# We'll run a Random Forest model and obtain an OpenML run-object. We can
+# see the evaluations recorded per fold for the dataset and the information
+# available for this run.
+
+run1 = openml.runs.run_model_on_task(model=clf, task=task)
+measures = run1.fold_evaluations
+
+# The timing and performance metrics available are
+for key in measures.keys():
+    print(key)
+
+# The performance metric is recorded under `predictive_accuracy` per fold
+# and can be retrieved as:
+for repeat, val1 in measures["predictive_accuracy"].items():
+    for fold, val2 in val1.items():
+        print("Repeat #{}-Fold#{}: {}".format(repeat, fold, val2))
+    print()
+
+# The remaining entries recorded in `measures` are the runtime records
+# related as:
+# usercpu_time_millis = usercpu_time_millis_training + usercpu_time_millis_testing
+# wall_clock_time_millis = wall_clock_time_millis_training + wall_clock_time_millis_testing
+#
+# The timing measures recorded as `*_millis_training` contain the per
+# repeat-fold timing incurred for the executing of the `.fit()` procedure
+# of the model. For `usercpu_time_*` the time recorded using `time.process_time()`
+# is converted to milliseconds and stored. Similarly, `time.time()` is used
+# to record the time entry for `wall_clock_time_*`. *_millis_testing`
+# follows the same procedure but for time taken for the `.predict()` procedure.
+
+# Comparing the CPU and wall-clock training times of the Random Forest model
+print_compare_runtimes(measures)
+
+############################################################################
+# Case 2: Running a Random Forest model on an OpenML task in parallel
+# *******************************************************************
+# Running the same model on the same task but with parallelism (n_jobs > 1)
```

Comment:
Could you please also give an example of setting `n_jobs` for the random forest? That should reduce both the wallclock and the cpu time, right? And could you please also highlight that fact?

Reply:
Done.